docker-bake.hcl

```diff
@@ -33,7 +33,7 @@ function "tags" {
 }
 
 group "default" {
-  targets = ["base", "full-stack", "lab"]
+  targets = ["base", "full-stack", "lab", "full-stack-lab"]
```

Comment:
I would very much prefer to name **this** image `full-stack`

Reply:
Ok, unless you object I will adopt the following image names:

- base: only contains AiiDA
- base-with-services: like base + services (installed and started)
- lab: base + AiiDAlab components
- full-stack: AiiDAlab + services